cmd/validator/flags/flags.go

```diff
@@ -19,6 +19,8 @@ const (
 	WalletDefaultDirName = "prysm-wallet-v2"
 	// DefaultHTTPServerHost for the validator client.
 	DefaultHTTPServerHost = "127.0.0.1"
+
+	DefaultMaxHealthChecks = 60
```

Comment:
```suggestion
	DefaultMaxHealthChecks = 0
```

I have issue with this. The default should match the existing behavior which was 0.

With a default of 60, there is new behavior where validators were voluntarily shut down rather than report unhealthy (infra should restart unhealthy nodes according to infra operators' configuration)

Reply:
Ok I am fine with that